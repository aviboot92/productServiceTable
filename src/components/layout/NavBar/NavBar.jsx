import React from 'react';
import {makeStyles} from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

const useStyles = makeStyles((theme) => ({
    root: {
        flexGrow: 1
    },
    logo: {
        maxWidth: 160,
        cursor: 'pointer'
    },
    title: {
        flexGrow: 1,
    }
}));

export default function ButtonAppBar(props) {
    const classes = useStyles();
    return (
        <div data-testid="navBar" className={classes.root}>
            <AppBar position='static'>
                <Toolbar>
                    <Toolbar>
                        <img
                            src={require("./../../../img/Banner Logo 280x60.png")}
                            alt="logo"
                            className={classes.logo}/>
                    </Toolbar>
                    <Typography variant="h6" className={classes.title}>
                        {props.label}
                    </Typography>
                </Toolbar>
            </AppBar>
        </div>
    );
}