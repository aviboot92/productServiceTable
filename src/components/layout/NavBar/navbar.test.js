import React from 'react';
import ReactDOM from 'react-dom';
import NavBar from './NavBar';
import {render} from '@testing-library/react';
import ReactTestRenderer from 'react-test-renderer';

it('should render without exploding', () => {
    const div = document.createElement("div");
    ReactDOM.render(<NavBar></NavBar>, div)
})

it('matches snapshot', () => {
    const tree1 = ReactTestRenderer.create(<NavBar label="Academy"></NavBar>).toJSON();
    expect(tree1).toMatchSnapshot();
})