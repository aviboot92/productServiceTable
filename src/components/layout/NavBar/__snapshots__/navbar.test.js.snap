// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="makeStyles-root-1"
  data-testid="navBar"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <img
          alt="logo"
          className="makeStyles-logo-2"
          src="Banner Logo 280x60.png"
        />
      </div>
      <h6
        className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
      >
        Academy
      </h6>
    </div>
  </header>
</div>
`;
